web/src/compose_ui.ts

```diff
@@ -399,6 +404,10 @@ export function handle_keydown(
         format_text($textarea, type);
         autosize_textarea($textarea);
         event.preventDefault();
+    } else if (type === "auto-surround") {
+        format_text($textarea, "auto-surround", key);
```

Comment:
The toggling behaviour doesn't really seem intuitive to me in this use case... If others concur, you can change that by calling `wrapFieldSelection` instead here